### DISCUSSION TITLE
手机端怎么显示菜单？

### DISCUSSION THREAD

**Discussion by Lukiya** [umijs/umi#8354] [Q&A]
使用umi max Ant Degisn Pro模板创建的项目。
pnpm dev启动后，手机访问页面时，左侧菜单无法展开。
发现一个按钮，但是点击无效。

应该怎么配置？

![image](https://user-images.githubusercontent.com/2961561/177007940-211d41ad-cb51-4207-b554-4598465f3698.png)

**Comment by fz6m** [comment]
pro layout 布局的问题去 https://github.com/ant-design/ant-design-pro 下面问吧
